paddlets/ensemble/ensemble_forecaster_base.py

```diff
@@ -152,7 +152,7 @@ def _predict_estimators(self,
             predictions.append(estimator.predict(tsdataset))
         return predictions
 
-    def save(self, path: str, ensemble_file_name: str = "paddlets-ensemble-partial.pkl") -> None:
+    def save(self, path: str, ensemble_file_name: Optional[str] = "paddlets-ensemble-partial.pkl") -> None:
```

Comment:
Line155 ensemble_file_name should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional